American-Elections-and-Crypto.md

```diff
@@ -0,0 +1,96 @@
+
+Introduction:
+
+Emphasizing the importance of the 2024 American elections and their potential ramifications for the global cryptocurrency landscape.
+
+Elections and Government Intervention:
+
+Questioning government influence: Analyzing the potential impact of the elections on government policies towards cryptocurrency and the cypherpunk ethos.
+Cypherpunk skepticism: Examining the cypherpunk perspective on government intervention in the crypto space and its implications for privacy and decentralization.
+
+Republican vs. Democratic Policies:
+
+Republican stance: Investigating potential Republican policies regarding cryptocurrency 
+Democratic perspective: Evaluating Democratic approaches to cryptocurrency 
+
+Global Implications and Sovereignty:
+
+Global ramifications: Exploring how decisions made in American politics can influence global perceptions of cryptocurrency and government regulation.
+Cypherpunk sovereignty: Discussing the importance of maintaining individual sovereignty and privacy rights in the face of government intervention and global regulatory trends.
+
+Navigating Cypherpunk Uncertainty:
+
+Strategies for cypherpunks: Providing insights and considerations for individuals who align with cypherpunk ideals in navigating the uncertainty of government intervention and regulatory changes.
+Upholding cypherpunk values: Encouraging readers to stay true to cypherpunk principles of privacy, decentralization, and individual sovereignty in their approach to cryptocurrency.
+
+---
+
+# Questioning the Future: American Elections and Crypto from a Cypherpunk Perspective
+
+The American elections of 2024 may be the most important event of the year. Not just to Americans, but to the whole world, since the US. remains the greatest economic and cultural power in our world. Scheduled for November 5, voters will determine both their legislative representatives and the future occupant of the presidency. The outcome of these elections is poised to reverberate across international boundaries, shaping the trajectory of global affairs for years to come.
+
+It appears that both the Republican and Democratic parties have solidified their contenders. With all indicators pointing towards a bid from the incumbent President Joe Biden representing the Democrats and the likelihood of former President Donald Trump stepping into the ring as the Republican nominee.
+
+Will the elections influence the crypto scene? Could crypto benefit from any candidate? And what does this mean for individuals deeply entrenched in the cypherpunk movement? Let's go through these questions!
+
+## Biden administration view on crypto
+
+The Biden administration was marked by a period of regulatory siege against the crypto market. These attacks were mainly coordinated by the Securities and Exchange Commission (SEC), which has Gary Gensler as its chairman, and increased after the bankruptcies of custodial service providers FTX, BlockFi, Voyager Digital, and other platforms.
```

Comment:
We need to find a way of separating the custodial crypto market from the Cypherpunk crypto market. 

We need a statement here that illustrates that the majority of the companies affected by this order had very little to do with real Cypherpunk principles and in many respects these orders helped to remove bad actors in the space. And although this had a negative reaction on all projects in the space those buildings peer to peer non-custodial products were mostly unaffected organisational and technically.

Reply:
added the statement!